src/api/request_models/administrator.py

```diff
@@ -11,6 +11,21 @@ class AdministratorAuthenticateRequest(RequestBase):
     password: SecretStr
 
 
+class AdministratorRegistrationRequest(RequestBase):
+    """Схема для регистрации администратора."""
+
+    name: str = Field(min_length=2, max_length=100)
+    surname: str = Field(min_length=2, max_length=100)
+    password: SecretStr
+
+    async def parse_to_db_obj(self, administrator: Administrator) -> Administrator:
+        administrator.name = self.name.title()

```

Comment:
```suggestion
    async def parse_to_db_obj(self) -> Administrator:
        administrator = Administrator()
        administrator.name = self.name.title()
```
Мне кажется, нет нужно передавать модель в аргументы метода